pom.xml

```
@@ -303,6 +303,23 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>net.revelc.code</groupId>
+        <artifactId>impsort-maven-plugin</artifactId>
+        <version>1.9.0</version>
+        <configuration>
+          <groups>java.,javax.,com.google.refine,org.openrefine,*</groups>
```

```suggestion
          <groups>java.,javax.,*,com.google.refine,org.openrefine</groups>
```

Let's use the order from the discussion to minimize the size of the diff.

---

I literally copied it from the discussion, where you wrote:

> java javax com.google.refine org.openrefine

But I'll change it to your new version.